meterfs/meterfs.c

```diff
@@ -48,6 +47,22 @@ static void meterfs_powerctrl(int state, meterfs_ctx_t *ctx)
 }
 
 
+int meterfs_fileNameCheck(const char *name)
+{
+	int i;
+
+	for (i = 0; i < MAX_NAME_LEN + 1; ++i) {
+		if (*(name + i) == '\0')
+			break;
+	}
+
+	if (i == 0 || i > MAX_NAME_LEN)
```

Comment:
```suggestion

	if ((*name == '\0' || strnlen(name, MAX_NAME_LEN + 1) > MAX_NAME_LEN)

```
It will be simpler to use the library function `strnlen()`
The function `meterfs_fileNameCheck()` can probably be of type `static`.